lib/webmock/http_lib_adapters/manticore_adapter.rb

```diff
@@ -58,7 +55,20 @@ def join_array_values(headers)
           end
 
           def response_object_for(client, request, context, &block)
-            request_signature = generate_webmock_request_signature
+            if request.respond_to?(:entity) && !request.entity.nil?
+              body = Manticore::EntityConverter.new.read_entity(request.entity)
+            else
+              body = nil
+            end
+
+            headers = request.headers.each_with_object({}) do |(k, v), h|
+              h[k] = case v
+                     when /,/ then  v.split(',').map(&:strip)
+                     else v
+                     end
+            end
```

Comment:
Sigh. So many header formatting things across the adapters in this project, and this is now the 2nd header-formatting method in this adapter alone...